_posts/2020-06-07-Human-Curation-and-Digital-Datasets.md

```diff
@@ -0,0 +1,213 @@
+copy the genealogies in the first instance.  Thus, it is often
+difficult to know as someone who is looking at the genealogies in the
+modern era what counts as an error and what counts as a motivated
+change to the tradition.  As [Eystein
+Thanisch](https://orcid.org/0000-0003-2819-5519) stated in [his
+article]({{site.baseurl}}/2020/06/06/Manuscript-Variants-and-the-Origins-of-Cíannachta-Glinne-Geimen.html)
+the policy of IrishGen is to take all statements in the manuscript
+tradition seriously thus the curators attempt to preserve what is
+extant rather than attempting to editorialise.
+
+## Ditigal Curation and Data Governance
+
+The first point is Digital Curation of historical material is not a
+new subject.  The [Digital Duration Centre](https://www.dcc.ac.uk/)
+(DCC) of the [University of Edinburgh](https://www.ed.ac.uk/) is a
+department which is entirely devoted to the curation and preservation
+of digital artefacts.  Additionally, the [International Journal of
+Digital Curation](http://www.ijdc.net/) is one forum for discussion of
+these issues.  Other fora can be found on the DCC's [publication
+website](https://www.dcc.ac.uk/publications/ijdc).
+
+The second point is that corporate Data Governance is also a well
+known discipline which attempts to curate and standardise data within
+a corporate environment. The Data Steward ensures that the data can be
+both stored and handled securely in terms of law
+(e.g. [HIPAA](https://www.hhs.gov/hipaa/index.html) in The United
+States and [GDPR](http://data.europa.eu/eli/reg/2016/679/oj) in the
+European Union) and used profitably by the corporation.
+
+Both of these disciplines have much in common and often overlap in
+terms and structure.  In terms of IrishGen, Digital Curation is
+probably more important but both disciplines emphasise having data
+policies which guide the collection and organisation of data
+artefacts.  As stated by Eystein Thanisch, the primary policy of
+IrishGen is to take the original data seriously, even if that means,
+when working with digital editions like those on
+[CELT](https://celt.ucc.ie/), going back to the original manuscript
+source to check on possible transcription errors originating from the
+translation of the information from manuscript to printed source to
+digital source.
+
+Unfortunately, due to lack of resources and lack of time on the
+current curator's part, the insights from these two disciplines have
+not been explored in detail.  When creating databases like IrishGen,
+having a curator or a willing volunteer from one of these disciplines
+can be a boon to disrupting patterns of error before they reach a
+point at which an end user, such as a scholar or other interested party,
+encounters the data.
+
+## Automation
+
+Unlike the scribes and redactors of early medieval Ireland, who mainly
+relied on repetition and structure to reduce errors, IrishGen relies
+on those to create automated tools to translate digital versions of
```

Comment:
"on those" = "on repetition and structure"? Maybe make this a bit clearer. E.g. "on the same features", or just repeat "on repetition and structure".

Reply:
Also, make it clear that, by "translate", you mean "encode" or suchlike.